lib/osf-components/addon/components/contributor-list/component.ts

```diff
@@ -71,6 +81,46 @@ export default class ContributorList extends Component {
         blocker.done();
     });
 
+    @task({ withTestWaiter: true, enqueue: true })
+    removeMeTask = task(function *(this: ContributorList) {
+        if (!this.node || this.node.isAnonymous || !this.currentUser.currentUserId) {
```

Comment:
What happens when this is on a no-project draft registration?

Reply:
@brianjgeiger  Some minor refactoring will be needed. We should define an interface that that defines the shape of the argument `node`, preferably also change the name.  The [invocation sites](https://github.com/CenterForOpenScience/ember-osf-web/pull/1129/files#diff-a9e01b74476d2d44581bb487750cda449e03e5d15d674ddcf399e3d8fca64c63R28) will have to be modified so we don't always go with `draftRegistration.branchedFrom`. Additionally, `draftRegistration` needs a `bibliographicContributors` relationship to work with contributor-list component. Does this fall under the scope of this ticket?

Reply:
No, this would probably fit under the integration ticket or a No Project ticket. We might want to make a ticket for Platform to add the `bibliographicContributors` relationship.

Reply:
> I would think for a no-project registration, `this.node` would be a DraftNode (we don't currently have that model on the FE), but I think this logic would still hold whether `this.node` is a Node or DraftNode
Ideally, contributor mgmt of a no-node draft should be performed on the draft itself, not the node behind it.